packages/client/modules/teamDashboard/components/AgendaListAndInput/AgendaListAndInput.tsx

```diff
@@ -31,30 +35,70 @@ interface Props {
   dashSearch?: string
   gotoStageId?: ReturnType<typeof useGotoStageId>
   isDisabled?: boolean
+  meeting: AgendaListAndInput_meeting | null
   team: AgendaListAndInput_team
-  meetingId?: string
+}
+
+const getAgendaItems = (meeting) => {
```

Comment:
can add the typing of `AgendaListAndInput_meeting | null` to meeting so you don't need the `meeting.phases!`